docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Share Process Namespace between Containers in a Pod
+min-kubernetes-server-version: v1.10
+approvers:
+- dawnchen
+- verb
+---
+
+{% capture overview %}
+
+{% include feature-state-alpha.md %}
+
+This page shows how to configure process namespace sharing for a pod. When
+process namespace sharing is enabled, processes in a container will be visible
+to all other containers in that pod.
+
+This can be useful for cooperating containers, such as a log handler sidecar
+container, or troubleshooting container images that don't include debugging
+utilities like a shell.
```

Comment:
I understand what you're saying here, but I had to read it a few times. Might be clearer (again, optional) with something like:
"You can use this feature to configure cooperating containers, such as " ...

Reply:
Done